python/lsst/meas/base/forcedPhotCcd.py

```diff
@@ -143,7 +143,7 @@ class ForcedPhotCcdConfig(ForcedPhotImageConfig):
     doApplyUberCal = lsst.pex.config.Field(
         dtype=bool,
         doc="Apply meas_mosaic ubercal results to input calexps?",
-        default=False
+        default=True
```

Comment:
I am not sure if this will eventually also apply to the `jointcal` calibration, but no, this currently won't work if there are no `meas_mosaic` results, which have only been run on HSC data (I'm not sure how difficult it would be to get it working on other cameras, nor if there is any real desire to put in that effort given that `jointcal` is intended to be the adopted `ubercal` code).

Note that this was your recommendation (and no one opposed)...are you having second thoughts?  This would mean any non `meas_mosaic`'d processing would need to add **doApplyUberCal=False** to their command lines (be it **makeCoaddTempExp.py** or **coaddDriver.py**) or, alternatively, as config overrides in their **config/makeCoaddTempExp.py** (which we could, in principle, do as part of this ticket).  Let me know if you lean in a particular direction.